src/main/java/de/unistuttgart/iste/gits/courseservice/persistence/dao/ChapterEntity.java

```diff
@@ -36,10 +38,7 @@ public class ChapterEntity {
     @Column(nullable = false)
     private OffsetDateTime endDate;
 
-    @ManyToOne
-    private CourseEntity course;
-
-    @ElementCollection
-    private List<UUID> contentIds;
+    @Column(nullable = false, name = "course_id")
+    private UUID courseId;
```

Comment:
Only having the uuid is enough, in cases where we have the id of the course, like when creating a new chapter for a course, this means that we don't have to query the course from the db but only set the id